### PR TITLE
add chad to the aur

### PR DESCRIPTION
**Pull request by ginglis13**
for all those arch heads out there 

https://aur.archlinux.org/packages/chad_stride/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,12 @@ make install
 make uninstall
 ```
 
+**AUR:**
+
+`yay -S chad_stride`
+
+*if you run into any issues with the AUR package, contact @ginglis13*
```
Comment (AndroidKitKat) [inline README.md]: Isn't this unneeded? Doesn't the [PKGBUILD](https://aur.archlinux.org/packages/chad_stride/) on the AUR already list the maintainer?

**Review by AndroidKitKat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AndroidKitKat) [reply README.md]: Also, yay is not standard for Arch Linux. You should just list the package name for on the AUR

Comment (ginglis13) [reply README.md]: you're right about it being unneeded. If it's that much of a bother, i can remove my name lol

Comment (ginglis13) [reply README.md]: also, you are right that there is no standard for an AUR package manager - but yay is the most common.

**Review by sambattalio — APPROVED** [review]
Ayyyyy I will let you know when you might have to bump the version #

**Pull request merged**